src/pyclaw/gauges.py

```diff
@@ -98,6 +98,10 @@ def read(self, gauge_id, path=None, use_pandas=False):
             # First line
             data = gauge_file.readline().split()
             self.id = int(data[2])
+            if len(data) == 8:
```

Comment:
That would be cleaner, but would require changes also to what is written out to `gauges.data`.  For now I suggest we just merge this simple change rather than risking breaking 2d and 3d somewhere.